runtime/cc-runtime.spec-template

```diff
@@ -21,6 +21,7 @@ Requires: cc-runtime-config
 Requires: qemu-lite
 Requires: clear-containers-image
 Requires: linux-container
+Requires: virtcontainers-pause
```

Comment:
The intention here is to remove completely `virtcontainers-pause` package and instead `cc-runtime` package should provide `cc-runtime` binary and `pause` binary